src/modules/m_nationalchars.cpp

```diff
@@ -276,7 +276,7 @@ class ModuleNationalChars : public Module
 		std::string casemapping = tag->getString("casemapping", FileSystem::GetFileName(charset), 1);
 		if (casemapping.find(' ') != std::string::npos)
 			throw ModuleException("<nationalchars:casemapping> must not contain any spaces!");
-		ServerInstance->Config->CaseMapping = casemapping;
+		ServerInstance->Config->CaseMapping = std::move(casemapping);
```

Comment:
The v3 branch has to build as GNU++03 so `std::move` can't be used. This can be committed to master though.

Reply:
@SadieCat, fix it.
Commit info here:
https://github.com/inspircd/inspircd/pull/1879/commits/b4e0c5b6212def02386af0a555445412eed6f1f4